Langs/0408.lang

```diff
@@ -1,95 +1,95 @@
+Done=Έτοιμο
 Loading WMIC=Φόρτωση WMIC
 Warning=Προειδοποίηση
-Your Computer is NOT ready for Windows 11, you can join the Discord using the Discord Icon if you need assistance.=Ο υπολογιστής σας δεν είναι συμβατός με τα Windows 11. Μπορείτε να συμμετάσχεται στο Discord, πατόντας το εικονίδιο του Discord, σε περίπτωση όπου χρείαζεσται βοήθεια.
-Your Computer is ready for Windows 11. You should receive the option to upgrade between October 5th 2021 and Fall 2022.=Ο υπολογιστής σου είναι συμβατός με τα Windows 11. Πιθανότητα θα λάβετε επιλογή αναβάθμισης, μεταξύ στις 5 Οκτωμβρίου του 2021 και του τέλος του 2022.
-Language Switcher currently disabled with Group Policy=Η αλλαγή γλώσσας έχει απενεργοποιηθει από την πολιτική ομάδας
-Theme Switcher currently disabled with Group Policy.=Η αλλαγή θέματος έχει απενεργοποιηθει από την πολιτική ομάδας.
-Default - No Theme Files Found=Προεπιλογή - Δεν εντοπίστηκαν αρχεία θεμάτων
+Your Computer is NOT ready for Windows 11, you can join the Discord using the Discord Icon if you need assistance.=Ο υπολογιστής σας ΔΕΝ είναι έτοιμος για τα Windows 11. Μπορείτε να συμμετάσχετε στο Discord, χρησιμοποιηώντας το σχετικό εικονίδιο, εάν χρειάζεστε βοήθεια. 
+Your Computer is ready for Windows 11 and its updates.=Ο υπολογιστής σας είναι έτοιμος για τα Windows 11 και τις ενημερώσεις του.
+Your Computer is ready for Windows 11 and its updates, but Windows Update may think you are not for 30 days. You can fix this using the Windows Installation Assistant.=Ο υπολογιστής σας είναι έτοιμος για τα Windows 11 και τις ενημερώσεις του, αλλά το Windows Update μπορεί να μην το πιστεύει, για 30 ημέρες. Μπορείτε να το επιδιωρθόσετε χρησιμοποιώντας τον Βοηθό Εγκατάστασης Των Windows.
+Your Computer is already on Windows 11. Would you like to check Feature Update Compatiblity instead?=Ο υπολογιστής σας ήδη τρέχει Windows 11. Θα θέλατε να ελέγξετε την συμβατότητα, για ενημερώσεις δυνατοτήτων;
+Language Switcher currently disabled with Group Policy=Ο Εναλακτής Γλώσσας είναι αυτήν την στιγμή απενεργοποιημένος, μέσω Group Policy.
+Theme Switcher currently disabled with Group Policy.=Ο Εναλακτής Θέματος είναι αυτήν την στιγμή απενεργοποιημένος, μέσω Group Policy.
+Default - No Theme Files Found=Προεπιλογή - Δεν βρέθηκαν αρχεία θέματος
 Guides=Οδηγοί
 Windows 11 Requirements=Απαιτήσεις των Windows 11
-Convert Disk to GPT=Μετατροπή δίσκου σε GPT
-Enable TPM=Ενεργοποίηση TPM
-Enable Secure Boot=Ενεργοποίηση Secure Boot
-Skip CPU && TPM=Παράλειψη επεξεργαστή + TPM
-Get Windows 11 Now=Λήψη των Windows 11 τώρα!
+Convert Disk to GPT=Μετατροπή δίσκου σε G.P.T.
+Enable TPM=Ενεργοποίηση T.P.M.	
+Enable Secure Boot=Ενεργ/ηση Ασφαλής Εκκίνησης
+Skip CPU && TPM=Παρ/μψη Απ/εων C.P.U. && T.P.M.
+Get Windows 11 Now=Λάβετε τα Windows 11, Τώρα!
+Cores=# Πυρήνες
+Threads=# Νήματα
+Not Supported=# Μη Υποστηριζόμενο
+May Report DirectX 12 Incorrectly=# Μπορεί να αναφέρει DirectX12, εσφαλμένα.

```

Comment:
Why change from `DirectX 12` to `DirectX12`? In other lines here you just changed opposite way.